packages/build_ext/build_ext/i18n.py

```diff
@@ -207,3 +209,18 @@ def run(self):
         # Delete the directory holding the temporary files created by intltool-extract
         # and the temporary keys.pot
         shutil.rmtree('tmp')
+
+
+class GettextWithOptParse(Gettext):
+    def find_py(self):
+        # Can't use super since we're descended from a old-style class
+        files = Gettext.find_py(self)
+
+        # We need to grab some strings out of optparse for translation
+        import optparse
```

Comment:
Module `optparse` is considered as deprecated. It should be replaced with `argparse` module.

Reply:
subscription-manager still uses optparse in modules like `managercli`.  We need to pull some strings out of the actual module so that they will go into our translation files.  If we switch subscription-manager to use argparse, we'd need to do something similar here, but until then we're going to need to import optparse here so we can run it through gettext.